packages/twenty-front/src/modules/activities/events/components/EventRow.tsx

```diff
@@ -1,11 +1,210 @@
+`;
+
+const StyledItemTitleContainer = styled.div`
+  display: flex;
+  flex: 1;
+  flex-flow: row ${() => (useIsMobile() ? 'wrap' : 'nowrap')};
+  gap: ${({ theme }) => theme.spacing(1)};
+  overflow: hidden;
+`;
+
+const StyledItemAuthorText = styled.div`
+  display: flex;
+  color: ${({ theme }) => theme.font.color.primary};
+  gap: ${({ theme }) => theme.spacing(1)};
+`;
+
+const StyledItemTitle = styled.div`
+  display: flex;
+  flex-flow: row nowrap;
+  overflow: hidden;
+`;
+
+const StyledItemTitleDate = styled.div`
+  align-items: center;
+  color: ${({ theme }) => theme.font.color.tertiary};
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(2)};
+  justify-content: flex-end;
+  margin-left: auto;
+`;
+
+const StyledVerticalLineContainer = styled.div`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(2)};
+  justify-content: center;
+  width: 26px;
+  z-index: 2;
+`;
+
+const StyledVerticalLine = styled.div`
+  align-self: stretch;
+  background: ${({ theme }) => theme.border.color.light};
+  flex-shrink: 0;
+  width: 2px;
+`;
+
+const StyledTooltip = styled(Tooltip)`
+  background-color: ${({ theme }) => theme.background.primary};
+
+  box-shadow: 0px 2px 4px 3px
+    ${({ theme }) => theme.background.transparent.light};
+
+  box-shadow: 2px 4px 16px 6px
+    ${({ theme }) => theme.background.transparent.light};
+
+  color: ${({ theme }) => theme.font.color.primary};
+
+  opacity: 1;
+  padding: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledTimelineItemContainer = styled.div<{ isGap?: boolean }>`
+  align-items: center;
+  align-self: stretch;
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(4)};
+  height: ${({ isGap, theme }) =>
+    isGap ? (useIsMobile() ? theme.spacing(6) : theme.spacing(3)) : 'auto'};
+  overflow: hidden;
+  white-space: nowrap;
+`;
+
+type EventRowProps = {
+  targetableObject: ActivityTargetableObject;
+  isLastEvent?: boolean;
+  event: Event;
+};
+
+export const EventRow = ({
+  isLastEvent,
+  event,
+  targetableObject,
+}: EventRowProps) => {
+  const beautifiedCreatedAt = beautifyPastDateRelativeToNow(event.createdAt);
+  const exactCreatedAt = beautifyExactDateTime(event.createdAt);
+
+  const properties = JSON.parse(event.properties);
+  const diff: Record<string, { before: any; after: any }> = properties?.diff;
+
+  const isEventType = (type: 'created' | 'updated') => {
+    return (
+      event.name === type + '.' + targetableObject.targetObjectNameSingular
+    );
+  };
 
-export const EventRow = ({ event }: { event: Event }) => {
   return (
     <>
-      <p>
-        {event.name}:<pre>{event.properties}</pre>
-      </p>
+      <StyledTimelineItemContainer>
+        <StyledIconContainer>
+          {isEventType('created') && <IconCirclePlus />}
+          {isEventType('updated') && <IconEditCircle />}
+          {!isEventType('created') && !isEventType('updated') && (
+            <IconQuestionMark />
```

Comment:
Maybe a normal icon could be a better default since people could eventually push other type of event through the API? IconFocusCenter or IconViewfinder?